packages/mco/components/data-policies/app-data-policies-status/data-policies-status-modal.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { Modal, Button, Nav, NavList, NavItem } from '@patternfly/react-core';
+import {
+  DRPoliciesType,
+  getDRPoliciesCount,
+} from '../../disaster-recovery/application-dr-status/dr-status-card';
+import { DRPoliciesStatusTable } from '../../disaster-recovery/application-dr-status/dr-status-table';
+import './data-policies-status-modal.scss';
+
+export const DataPoliciesStatusModal: React.FC<DataPoliciesStatusModalProps> =
+  ({ isOpen, dataPoliciesStatus, close }) => {
+    const { t } = useCustomTranslation();
+    const policiesCount = getDRPoliciesCount(dataPoliciesStatus?.drPolicies);
+    const [selectedIndex, setSelectedIndex] = React.useState(0);
+    const navItems = [
+      {
+        name: t('Disaster Recovery ({{count}})', { count: policiesCount }),
+        component: (props) => <DRPoliciesStatusTable drPolicies={props} />,
```

Comment:
```suggestion
        component: DRPoliciesStatusTable,
```
Use this directly as `<component {...props}/>` where required.